content/en/docs/Writing policies/generate.md

```diff
@@ -246,7 +246,107 @@ spec:
           foo: bar
 ```
 
-## Generating resources into existing namespaces
+## Generate for Existing resources
+
+With Kyverno 1.7.0+, Kyverno supports the generate for existing resources. Generate existing policies are applied in the background which creates target resource in existing matched resources in the cluster.
+They may also optionally be configured to apply upon updates to the policy itself.
```

Comment:
No need for a new paragraph here. Put this sentence with the previous paragraph.